FAQ/FAQ-Apps.md

```diff
@@ -6,6 +6,7 @@ This document covers frequently asked question about the Apps in BTCPay.
 * [Is there a limit on the number of Apps I can deploy?](#is-there-a-limit-on-the-number-of-apps-i-can-deploy)
 * [Is there a Point of Sale feature in BTCPay?](#is-there-a-point-of-sale-feature-in-btcpay)
 * [How can I use BTCPay in a physical store?](#how-can-i-use-btcpay-in-a-physical-store)
+* [How to customize the apperance of POS in BTCPay?](#how-to-customize-the-appearance-of-POS-in-BTCPay)
```

Comment:
Well, it's a common abbreviation in commerce. And I think it's quite clear since all above questions refer it as POS - Point of Sale app. But I fixed it.